tests/subsys/fs/common/test_fs_open_flags.c

```diff
@@ -146,6 +146,10 @@ void test_fs_open_flags(void)
 	ZOPEN(&ts, FS_O_WRITE, -ENOENT);
 	ZOPEN(&ts, FS_O_READ, -ENOENT);
 	ZOPEN(&ts, FS_O_RDWR, -ENOENT);
+	/* This is valid case for API, but FAT driver will always create,
```

Comment:
Comment is out of date (at least by referencing `FO_`.  It's also very unclear: it suggests that FatFS is different that other file systems here; if it was, it wouldn't return `-ENOENT` so the test would fail, right?

Clarification is needed.  Best alternative is this comment is removed because it's no longer necessary.

Reply:
Ah; I see the name at least gets fixed in patch 9.  Please move the fix to where the comment is introduced (or remove the comment there).

Reply:
Will fix. The commits in series should not fix each other, so there is mistake.

Reply:
Fixed.